NuGet.config

```diff
@@ -4,6 +4,9 @@
   <packageSources>
     <clear />
     <!--Begin: Package sources managed by Dependency Flow automation. Do not edit the sources below.-->
+    <add key="darc-pub-dotnet-core-setup-7d57652" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-core-setup-7d57652f/nuget/v3/index.json" />
+    <add key="darc-pub-dotnet-corefx-4ac4c03" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/darc-pub-dotnet-corefx-4ac4c036/nuget/v3/index.json" />
+    <add key="darc-int-dotnet-core-setup-a2c3406" value="https://pkgs.dev.azure.com/dnceng/_packaging/darc-int-dotnet-core-setup-a2c3406f/nuget/v3/index.json" />
```

Comment:
Remove this. There was mail about having to remove darc-int-* manually due to an issue being worked on.

Reply:
(Separate, but also wish it wouldn't add the feeds that are redundant with nuget.org as dar-pub-*)

Reply:
@nguerrera That's what his feed cleanup is going to do.  It will recognize that a 3.0.0 package or whatever is on nuget.org or other permanant feed.  and the dep update will stop adding the special feed to nuget.config

Reply:
Yay!

Reply:
@sfoslund If you see a similar failure, remove the internal feed

Reply:
@mmitche do you know the progress of the extra darc-int? I have to remove them all over the place. I made about 10 PRs now.